site/plugins/buy/Price.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Buy;
+
+use Kirby\Data\Data;
+
+class Price
+{
+	public readonly float $rate;
+
+	public function __construct(
+		public readonly Product $product,
+		public readonly string $currency = 'EUR'
+	) {
+		$rates = Data::read(__DIR__ . '/rates.json');
+		$this->rate = $rates[$currency];
```

Comment:
This will explode for an invalid currency.

Reply:
The JSON entries and the list of Paddle currencies need to be  the same anyways, so that's not a case that needs to be supported.

Reply:
Added a fallback to EUR that also changes sets `currency` to EUR then